integrations/clickup/main.py

```diff
@@ -0,0 +1,74 @@
+from enum import StrEnum
+from typing import Any
+
+from loguru import logger
+from clickup.client import ClickupClient
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+
+async def setup_application() -> None:
+    logic_settings = ocean.integration_config
+    app_host = logic_settings.get("app_host")
+    if not app_host:
```

Comment:
You aren't making use of app_host in your code. No reason it should be here.

Reply:
done